packages/tax/integration-tests/__tests__/index.spec.ts

```diff
@@ -8,6 +8,53 @@ moduleIntegrationTestRunner({
   moduleName: Modules.TAX,
   testSuite: ({ service }: SuiteOptions<ITaxModuleService>) => {
     describe("TaxModuleService", function () {
+      it("should create tax rates and update them", async () => {
+        const region = await service.createTaxRegions({
+          country_code: "US",
+          default_tax_rate: {
+            name: "Test Rate",
+            rate: 0.2,
+          },
+        })
+
+        const rate = await service.create({
+          tax_region_id: region.id,
+          name: "Shipping Rate",
+          code: "test",
+          rate: 8.23,
+        })
+
+        await service.update(rate.id, {
+          name: "Updated Rate",
+          code: "TEST",
+          rate: 8.25,
+        })
+
+        await service.update(
```

Comment:
It's nice to check that the returned value from `update` and `create` matches what we expect, i.e the model that got created/updated